pkg/actuators/machine/actuator.go

```diff
@@ -370,31 +371,27 @@ func (a *Actuator) Update(context context.Context, cluster *clusterv1.Cluster, m
 		if err := a.updateStatus(machine, nil); err != nil {
 			return err
 		}
-
-		errMsg := "attempted to update machine but no instances found"
-		glog.Error(errMsg)
-		return fmt.Errorf(errMsg)
+		return nil
```

Comment:
This was previously an error, but it's not now?

Reply:
We could return error here.  I'm in favor returning nil.  If we return error, this is going to requeue perpetually until the machine-object is deleted (unless someone creates and instance outside of the machine-controller and tags it with the appropriate name, which I don't think we want to encourage).

In any case, we should never arrive at this logic during normal operation.

Reply:
Kind of seems like an error should be returned. Not sure it should be framed strictly in terms of re-queuing in the controller. Just in terms of the actuator API, if I call `Update()` and the instance doesn't exist, I would expect to get an error.